Purchases/Public/RCEntitlementInfo.h

```diff
@@ -120,8 +89,11 @@ NS_SWIFT_NAME(Purchases.EntitlementInfo)
  or shared to them by a family member. This can be useful for onboarding users who have had
  an entitlement shared with them, but might not be entirely aware of the benefits they now have.
  */
-@property (readonly) RCPurchaseOwnershipType ownershipType;
+@property (readonly) enum RCPurchaseOwnershipType ownershipType;
```

Comment:
I'm curious, what does `enum` do in this context? is it needed because we're no longer doing the import?

Reply:
It's a forward declaration for enums 🙃
It's needed exactly as you guessed 😄

Reply:
Awesome! I've been looking for ways to do this and couldn't come up with results for some reason.